app/src/main/res/values-en/strings.xml

```diff
@@ -925,24 +925,24 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_kerb_height_lowered_ramp">Curb ramp</string>
     <string name="quest_kerb_height_no">No curb at all</string>
 
-    <string name="quest_lanes_title">"How many lanes for cars does this road have?"</string>
+    <string name="quest_lanes_title">"How many car lanes does this road have?"</string>
     <string name="quest_lanes_answer_lanes">"Marked lanes"</string>
-    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs for each side"</string>
-    <string name="quest_lanes_answer_lanes_description2">"Specify the total number of marked lanes (both directions) usable by car traffic. So, bike and dedicated parking lanes don’t count.
+    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs on each side"</string>
+    <string name="quest_lanes_answer_lanes_description2">"Specify the total number of marked lanes (both directions) usable by car traffic. Bike and dedicated parking lanes don’t count.
 If any lanes are reserved for buses, please leave a note instead."</string>
-    <string name="quest_lanes_answer_lanes_description_one_side2">"Specify the number of marked lanes usable by car traffic here. So, bike and dedicated parking lanes don’t count.
+    <string name="quest_lanes_answer_lanes_description_one_side2">"Specify the number of marked lanes usable by car traffic here. Bike and dedicated parking lanes don’t count.
 If any lanes are reserved for buses, please leave a note instead."</string>
     <string name="quest_lanes_answer_noLanes">"No marked lanes"</string>
     <string name="quest_lanes_answer_lanes_center_left_turn_lane">"It has a two-way left turn lane"</string>
 
-    <string name="quest_laundrySelfService_title2">Does this laundry provide self-service?</string>
+    <string name="quest_laundrySelfService_title2">Is this a self-service laundromat?</string>
 
     <string name="quest_leafType_title">Do the trees here have needles or leaves?</string>
     <string name="quest_leaf_type_needles">Needles</string>
-    <string name="quest_leaf_type_broadleaved">Broadleaved</string>
+    <string name="quest_leaf_type_broadleaved">Leaves</string>
```

Comment:
Needles technically are leaves

Reply:
The question is literally "Do the trees here have needles or leaves?", I'd say in the context "leaves" is less confusing and doesn't confuse a user that is looking at leaves that are smaller.

Reply:
Also, "broadleaved" is not a term I've ever seen in US English and I always assumed it to be an OSM-ism

Reply:
Okay, makes sense. I'll resolve this discussion and mine too (I noted the same further below)

Reply:
The distinction I learned in fifth grade science class was between conifers (`needles`) and broadleaf trees (`broadleaved`), with ferns throwing a wrench into this whole classification scheme.